src/main/docs/schemas/pet-schema.ts

```diff
@@ -8,6 +8,8 @@ export const petSchema = {
     specie: {
       $ref: '#/schemas/specie'
     },
-    specieAlias: { type: 'string', nullable: true }
+    specieAlias: { type: 'string', nullable: true },
+    petName: { type: 'string' },
+    gender: { type: 'string' }
```

Comment:
Same that here: 
https://github.com/PetJournal/petjournal.api/pull/44#discussion_r1372019338